e2e-tests/cypress/tests/integration/channels/messaging/emoji_picker_keyboard_usability_spec.js

```diff
@@ -117,10 +114,10 @@ describe('MM-13064 - Emoji picker keyboard usability', () => {
         cy.findByLabelText('Search for an emoji').should('have.value', 'sm');
 
         // # Move cursor to the beginning then end of search text, type "i", and then verify search text input
-        cy.findByLabelText('Search for an emoji').type('{uparrow}{downarrow}i').should('have.value', 'smi');
+        cy.findByLabelText('Search for an emoji').type('{uparrow}{uparrow}{downarrow}i').should('have.value', 'smi');
```

Comment:
@M-ZubairAhmed , since the emoji have focus when the up/down keys are pressed, we have press the keys to navigate through 'emoji searching text'. Just getting your inputs if this is expected behaviour or need to be changed in app.

Reply:
Yes that's correct we navigate emojis on keyboard and after reaching top row it goes to search